core/trino-main/src/main/java/io/trino/operator/scalar/SqlFunctions.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator.scalar;
+
+import com.google.common.base.MoreObjects;
+import io.airlift.slice.Slice;
+import io.trino.spi.TrinoException;
+import io.trino.spi.function.Description;
+import io.trino.spi.function.ScalarFunction;
+import io.trino.spi.function.SqlType;
+import io.trino.spi.type.StandardTypes;
+import io.trino.sql.parser.ParsingException;
+import io.trino.sql.parser.SqlParser;
+import io.trino.sql.tree.NodeLocation;
+import io.trino.sql.tree.Statement;
+
+import static io.airlift.slice.Slices.utf8Slice;
+import static io.trino.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
+import static io.trino.sql.SqlFormatter.formatSql;
+
+public final class SqlFunctions
```

Comment:
nit: `SqlFunctions` does not ring a bell that those are functions that operate on SQL text (at least it did not for me). 
Maybe just name the class `FormatTrinoQueryFunction`. Do you expect to add more here?